### PR TITLE
Add constants for color strings

### PR DESCRIPTION
**Pull request by fisher-alice**
This is a quick follow-up to https://github.com/code-dot-org/code-dot-org/pull/55007 which was blocking curriculum from making HoC screencast videos for the video team.

This PR adds constants for color strings that were updated in the prior PR.

## Links
[trello](https://trello.com/c/gTzUI8Kd/182-follow-up-to-instructions-update-cleanup-refactor-of-color-constants)

## Testing story
Tested locally to confirm no changes based on this PR.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by thomasoniii — APPROVED** [review]
lgtm 🌈 🚀

File: apps/src/templates/instructions/ChatBubble.jsx [hunk 1]
```diff
@@ -51,6 +52,8 @@ var audioStyle = {
   },
 };
 
+const MINECRAFT_COLOR = '#3B3B3B';
```
Comment (levadadenys) [inline apps/src/templates/instructions/ChatBubble.jsx]: I see that we have `$very-dark-gray` in `apps/style/craft/style.scss`. There's no easy way to import that file in this one. Hovewer, I guess we can rename it something like `MINECRAFT_VERY-DARK-GRAY-COLOR` for consistency.

**Review by levadadenys — APPROVED** [review]
LGTM! Thanks for updating this! One suggestion, but it's non-blocking.

File: apps/src/templates/instructions/ChatBubble.jsx (hunk 1 above)
Comment (fisher-alice) [reply apps/src/templates/instructions/ChatBubble.jsx]: Will do - thanks!

File: apps/src/templates/instructions/InstructionsCsfMiddleCol.jsx [hunk 2]
```diff
@@ -14,6 +14,8 @@ import color from '../../util/color';
 import Instructions from './Instructions';
 
 var instructions = require('../../redux/instructions');
+const VERY_LIGHT_BLUE_COLOR = '#f0ffff';
+const VERY_LIGHT_YELLOW_COLOR = '#fffff0';
```
Comment (fisher-alice) [inline apps/src/templates/instructions/InstructionsCsfMiddleCol.jsx:18]: @levadadenys - I ended up tweaking these variable names as well.

**Review by levadadenys — APPROVED** [review]

**Pull request merged**